llvm/test/MC/AArch64/armv9.4a-gcs.s

```diff
@@ -1,115 +1,205 @@
-// RUN: llvm-mc -triple aarch64 -mattr +gcs -show-encoding %s | FileCheck %s
+// RUN: llvm-mc -triple=aarch64 -show-encoding -mattr=+gcs < %s \
+// RUN:        | FileCheck %s --check-prefixes=CHECK-ENCODING,CHECK-INST
+// RUN: not llvm-mc -triple=aarch64 -show-encoding < %s 2>&1 \
```

Comment:
Can we change this  CHECK-ERROR to be NO-GCS. 
So you can remove line 14

Reply:
Sure, done.